src/Strategy/Python/PDM/Pdm.hs

```diff
@@ -0,0 +1,95 @@
+module Strategy.Python.PDM.Pdm (
```

Comment:
Could you add a short comment describing the relationship of this module to `Strategy.PDM`? The file structure suggests that PDM is a strategy in its own right, but this file's placement suggests it's actually part of the Python strategy. 

Another possibility could be to move this to `Strategy/PDM` if `PDM` should be considered its own strategy distinct from the `Python` strategy.

Reply:
Aha - I was following existing convention i.e. poetry (under python), gopkg (under golang), cabal/stack etc. Strategies are coupled with package manager, as opposed to languages. Let me know if you feel strongly about this - happy to change the layout/structure.

Reply:
addressed in https://github.com/fossas/fossa-cli/pull/1214/commits/fb13068ee19125441037fc138e26751d3b8073b3.